Dockerfile

```diff
@@ -11,31 +40,21 @@ COPY doc/datasets/popularize_churn.sql \
 VOLUME /var/lib/mysql
 
 # Install the Python source code.
-# TODO(yi): It seems that we don't need to build python/couler into a wheel.
```

Comment:
Thanks to the answer from @Yancey1989 , we do need to build Couler into a pip package, so we can get Couler dependencies when installing the package.